doc/ca65.sgml

```diff
@@ -2648,11 +2683,6 @@ Here's a list of all control commands and a description, what they do:
     overridden. When using this feature, you may also get into trouble if
     later versions of the assembler define new keywords starting with a dot.
 
-  <tag><tt>loose_char_term</tt><label id="loose_char_term"></tag>
-
-    Accept single quotes as well as double quotes as terminators for char
-    constants.
-
```

Comment:
Why did you remove this section? Looking at feature.c it seems that `loose_char_term`is present.